scripts/ci/push.sh

```diff
@@ -9,3 +9,4 @@ set -o errexit -o pipefail
 node ./scripts/await-in-progress.js
 ./scripts/ci/run-pulumi.sh update
 ./scripts/ci/make-s3-redirects.sh
+./scripts/update-search-index.sh
```

Comment:
Could we do this before the S3 redirect step? Only because this can take a while.

It does make me wonder whether we could do this before the Pulumi run, though. Couldn't we?

Reply:
yeah we definitely could. good call!